source/accessible-media/formats.html.erb.md

```diff
@@ -15,79 +14,38 @@ contributors:
   - The Education and Outreach Working Group (<a href="https://www.w3.org/WAI/EO/">EOWG</a>)
 support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/">U.S. Access Board, WCAG TA Project</a>
 ---
-## Caption formats
+Nearly all modern browsers and media players support displaying closed captions. However,
+they do not all support the same caption file formats. The most commonly used formats for online media are:
 
-Nearly all modern user agents (browsers, media players) on both desktop
-and mobile platforms provide support for the display of closed captions,
-but not all support the same caption-delivery formats. The three most
-common formats used for online media are listed below.
+-   [Web Video Text Tracks (WebVTT)](https://www.w3.org/TR/webvtt1/) <br>
```

Comment:
Changed order to have the most supported format on top. Inlining the browser support from the table into the list.